db/model/bot.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * db/model/bot.js
+ */
+const common = require('../helpers/common');
+const constants = require('../constants');
+
+const assoc = {};
+
+module.exports = function user(seq, dataTypes) {
+  const Bot = seq.define('Bot', {
+    name: {
+      type: dataTypes.STRING,
+      allowNull: false,
+      unique: true,
```

Comment:
any validation on names? do you allow special characters (symbols? spaces? etc.)? string length limit? is it case sensitive? (this comment applies to all models)